sympy/stats/crv_types.py

```diff
@@ -2317,8 +2317,33 @@ def check(mean, std):
     def pdf(self, x):
         return exp(-(x - self.mean)**2 / (2*self.std**2)) / (sqrt(2*pi)*self.std)
 
-    def sample(self):
-        return random.normalvariate(self.mean, self.std)
+    def sample(self, **kwargs):
+        mean, std = self.mean, self.std
+        print("len: ", len(kwargs))
+        if len(kwargs) == 0 or mean.has(Symbol) or std.has(Symbol):
+            return random.normalvariate(self.mean, self.std)
+        objs = {}
+        mean, std, size = float(mean), float(std), kwargs.get('size', 1)
+        if kwargs.get('numpy', False):
+            objs['numpy'] = _sample_numpy(mean, std, size)
```

Comment:
Don't you intend to return these?

Reply:
Yes. I will make the correction.

Reply:
If you are calling a class method, you should prepend `self.` to the method name.